app/src/main/java/hu/vmiklos/plees_tracker/SharedPreferencesLIveData.kt

```diff
@@ -0,0 +1,48 @@
+package hu.vmiklos.plees_tracker
+
+import android.content.SharedPreferences
+import androidx.lifecycle.LiveData
+
+/**
+ * From https://gist.github.com/rharter/1df1cd72ce4e9d1801bd2d49f2a96810#gistcomment-3541759
+ */
+private class SharedPreferenceLiveData<T>(
```

Comment:
I originally attempted to use a `MutableLiveData` in `MainViewModel` getting set from `SharedPreferencesChangeListener.onSharedPreferenceChanged` as suggested, but for whatever reason it never triggered the observer callback in `MainActivity`, so I opted for this other solution I found via https://stackoverflow.com/questions/50649014/livedata-with-shared-preferences